File: app/src/main/java/de/metzgore/beansplan/util/di/components/RefreshableDailyScheduleFragmentSubComponent.java
package de.metzgore.beansplan.util.di.components;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import de.metzgore.beansplan.dailyschedule.RefreshableDailyScheduleFragment;

@Subcomponent
public interface RefreshableDailyScheduleFragmentSubComponent extends
        AndroidInjector<RefreshableDailyScheduleFragment> {
    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<RefreshableDailyScheduleFragment> {
    }
}

File: app/src/main/java/de/metzgore/beansplan/util/di/components/RefreshableDailyScheduleFragmentSubComponent.kt
package de.metzgore.beansplan.util.di.components

import dagger.Subcomponent
import dagger.android.AndroidInjector
import de.metzgore.beansplan.dailyschedule.RefreshableDailyScheduleFragment

@Subcomponent
interface RefreshableDailyScheduleFragmentSubComponent : AndroidInjector<RefreshableDailyScheduleFragment> {
    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<RefreshableDailyScheduleFragment>()
}
